File: AkitaModel/Robin/Reports/LaboratoryResults.cs
using System.Collections.Generic;

namespace Skyware.Lis.AkitaModel.Robin.Reports;

/// <summary>
/// Data model for report "Laboratory results". 
/// </summary>
public class LaboratoryResults
{
    /// <summary>
    /// Sale/Visit data.
    /// </summary>
    public Sale Visit { get; set; }

    /// <summary>
    /// Patient data
    /// </summary>
    public Patient Patient { get; set; }

    /// <summary>
    /// Doctors data
    /// </summary>
    public IList<Doctor> ReferringDoctors { get; set; } = [];

    /// <summary>
    /// List of results
    /// </summary>
    public IList<SaleItem> Results { get; set; } = [];

    /// <summary>
    /// Footnotes list.
    /// </summary>
    public IList<Footnote> Footnotes { get; set; } = [];

}

File: AkitaMotelTS/src/Robin/Reports/LaboratoryResults.ts
/**
 * Skyware.Lis.AkitaModel.Robin.Reports
 */

import { Sale } from '../../Sale';
import { Patient } from '../../Patient';
import { Doctor } from '../../Doctor';
import { SaleItem } from '../../SaleItem';
import { Footnote } from '../../Footnote';

/**
 * Data model for report "Laboratory results".
 * @deprecated Use Results.VisitResults instead.
 */
export class LaboratoryResults {
  /**
   * Sale/Visit data.
   */
  visit?: Sale;

  /**
   * Patient data
   */
  patient?: Patient;

  /**
   * Doctors data
   */
  referringDoctors: Doctor[] = [];

  /**
   * List of results
   */
  results: SaleItem[] = [];

  /**
   * Footnotes list.
   */
  footnotes: Footnote[] = [];
}
